config.py

```diff
@@ -60,7 +60,7 @@ class ProductionConfig(Config):
     @classmethod
     def init_app(cls, app):
         Config.init_app(app)
-
+        assert os.environ.get('SECRET_KEY'), 'SECRET_KEY IS NOT SET!'
```

Comment:
Is there any way we could change the `SECRET_KEY =` logic to print out a warning if the secret key wasn't explicitly set and to fail if its a production config?